### ISSUE TITLE
SPIKE: Remove wallet requirements in swapper

### ISSUE THREAD

**Issue by gomesalexandre** [shapeshift/web#7865]
### Overview

Swapper is now accessible without a wallet connected:

<img width="1027" alt="image" src="https://github.com/user-attachments/assets/d76733d1-9fad-45fb-a7c3-dd44db770b8d">

However, it isn't possible to get quotes without a wallet connected, which makes the experience suboptimal.

This ticket serves as a spike to figure out whether or not we *could* get quotes upstream without a wallet, as well as the technical effort required here.

Time cap: 1 day

### References and additional details

- Which swapper/s can we get *rates* (as opposed to quotes, without sending in an address) for?
- If it is confirmed most/all swappers do support getting *rates* without an address sent in payload, how much of a lift is it / which pieces do we need to update to accommodate for that? 
- Do we want to clearly distinguish between *rate* and *quote* step? We already do that in swapper, though not for all swappers, and without a consistent vernacular/abstraction around this

### Acceptance Criteria

- We know which swappers/endpoints allow us to get a quote without an address passed in
- There is a rough estimation of effort/places which should be updated to get quotes without a wallet connected
- tickets created for next steps / implmentation

### Estimated effort

1 day

**Comment by gomesalexandre** [comment]
Spike 1 (swappers which support rates without a wallet connected): https://gist.github.com/gomesalexandre/b1cb6d3a90019e570ecfa1d3a74a193d

All of our swappers support this 🎉 

tl;dr: we will need to split the logic between rates (price, possibly with network fees, and without a send/receive address needed, though passed *if* available) and quote (always with send/receive address). We already do this for ZRX, and should follow the same pattern everywhere else.

**Comment by gomesalexandre** [comment]
Spike 2: remove bip44 requirements: https://gist.github.com/gomesalexandre/750cbb8f829eb7600129aef477dbdcac

This can be done either before or after accomodating swappers and splitting between trade quotes / trade rates, though should ideally be done first.

**Comment by gomesalexandre** [comment]
Closing as spike finished and two issues have been created:

https://github.com/shapeshift/web/issues/7940
https://github.com/shapeshift/web/issues/7941
